cpp/cmake/thirdparty/CUDF_GetCUCO.cmake

```diff
@@ -0,0 +1,37 @@
+#=============================================================================
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+
+function(find_and_configure_cuco)
+  if(CUCO_INCLUDE)
+    set(CUCO_INCLUDE_DIR "${CUCO_INCLUDE}" PARENT_SCOPE)
+    return()
+  endif()
+  if(CUCO_INCLUDE_DIR)
+    set(CUCO_INCLUDE_DIR ${CUCO_INCLUDE_DIR} PARENT_SCOPE)
+    return()
+  endif()
+  CPMFindPackage(NAME cuco
+    GITHUB_REPOSITORY PointKernel/cuCollections
```

Comment:
Should this be NVIDIA/cuCollections?

Reply:
The multimap branch hasn't been merged into the NVIDIA repo yet. Will update this once all reviews/issues in the current PR are addressed.